### ISSUE TITLE
updateByQuery in 7.0.1

### ISSUE THREAD

**Issue by rwoods02** [sksamuel/elastic4s#1727]
When I attempt to updateByQuery in version 7.0.1

```
       client.execute {
          updateByQuery(index, termQuery("_id", identifier))
            .script(script(s"ctx._source.fun = somethingFun;").lang("painless"))
            .refresh(RefreshPolicy.NONE)
        }.onComplete(response =>
          if (response.isFailure) {
          logger.error("WE DIDNT UPDATE" + response.get.error.reason, response.get.error)
          }
```

I receive this error 

```
org.elasticsearch.client.RestClient logResponse
WARNING: request [POST http://localhost/{index}/_doc/_update_by_query?refresh=false] returned 1 warnings: [299 Elasticsearch-7.0.1-e4efcb5 "[types removal] Specifying types in search requests is deprecated."]
``` 
As you can see I am not specifying a type but my request returns 1 because types are deprecated. I looked through your test cases and it seems like this is the correct syntax. How do I properly user updateByQuery

**Comment by sksamuel** [comment]
I think we just need to remove the _doc part so it doesn't use types anymore.

**Comment by sksamuel** [comment]
Fixed in 7.0.2
